app/styles/phone-number-input.less

```diff
@@ -0,0 +1,48 @@
+.phone-number-container {
+  position: relative;
+
+  .upf-infinite-select {
+    max-width: 100%;
+    .padding-xx-sm;
+
+    .upf-infinite-select__item {
+      padding: 0;
+      .padding-top-xxx-sm;
+      .padding-bottom-xxx-sm;
+    }
+
+    .row-selected span {
+      .text-color-default;
+    }
+  }
+
+  .phone-number-input {
+    .country-selector {
+      border-right: 1px solid @upf-gray;
+      height: 100%;
+      display: flex;
+      align-items: center;
+      padding-right: 12px;
+      margin-right: 12px;
```

Comment:
```suggestion
      padding-right: @spacing-xx-sm;
      margin-right: @spacing-xx-sm;
```

Reply:
That's not the same width sadly, I tried using our variables but couldn't get something that matched exactly.

Reply:
Ok ok no problem, it's good for me so